src/melee/ft/chara/ftFox/ftFox_SpecialN.c

```diff
@@ -153,40 +153,48 @@ void ftFox_RemoveBlaster(HSD_GObj* fighter_gobj)
     }
 }
 
-extern void func_8029C6A4(f32, f32, HSD_GObj*, Vec3*, s32);
-extern void func_802AE1D0(HSD_GObj*);
-
 u32 foxSFX[2] = { 0x1AE17, 0x1AE1A };
 u32 falcoSFX[2] = { 0x18703, 0x18706 };
 
 // 0x800E5F28
 // https://decomp.me/scratch/iPgDc // Create Blaster Shot Item
 void ftFox_CreateBlasterShot(HSD_GObj* fighter_gobj)
 {
+    /** @todo Shared @c inline with #ftFox_SpecialNLoop_Anim and
+     * #ftFox_SpecialAirNLoop_Anim.
+     */
     Vec3 sp2C;
     s32 unused;
     ftFoxAttributes* foxAttrs;
     Fighter* fp;
     f64 launchAngle;
     s32 ftKind;
 
+#ifdef MUST_MATCH
+    /// @todo Seems fake, probably one or more missing @c inline functions.
     fp = fp = getFighter(fighter_gobj);
+#else
+    fp = getFighter(fighter_gobj);
+#endif
```

Comment:
This only exists because the original line generates a warning in `clang`. I think it's pretty reasonable to aim for no warnings/errors.